src/main/webapp/resources/js/pages/admin/chart-config.js

```diff
@@ -41,15 +48,10 @@ export function getChartConfiguration(chartType, data) {
     width: "100%",
     height: chartHeight,
     meta: { key: { alias: "" }, value: { alias: "" } },
-    label: {
-      visible: Boolean(data),
-      position: "middle",
-      adjustColor: true,
-      style: { fill: "#0D0E68", fontSize: 12, fontWeight: 600, opacity: 0.3 },
-    },
     legend: {
       visible: Boolean(data),
       position: "bottom",
+      offsetY: 10
```

Comment:
Can I get you to add `maxWidth: window.innerWidth - 600` in the legend section as well. That way the legend doesn't extend from the chart margin on the left to the right

Reply:
Please see 8c5ca22cc7d591a5d0a7bb21f6559dba0b638564.